src/cmd_context/tactic_cmds.cpp

```diff
@@ -165,7 +165,23 @@ class exec_given_tactic_cmd : public parametric_cmd {
     }
 };
 
-typedef simple_check_sat_result check_sat_tactic_result;
+struct check_sat_tactic_result : public simple_check_sat_result {
+public:
+  svector<symbol> labels;
+
+  check_sat_tactic_result(ast_manager & m) : simple_check_sat_result(m) {
+  }
+
+  virtual void get_labels(svector<symbol> & r) {
+    r.append(labels);
+  }
+
+  virtual void add_labels(svector<symbol> & r) {
+    labels.append(r);
+  }
+};
+
+typedef svector<symbol> & labels_ref;
```

Comment:
Using labels_ref seems nicer than svector<symbol> throughout. Having class labels_ref : public svector<symbol> {}; furthermore allows to maintain that the vector of symbols is related to labels.